pyproject.toml

```diff
@@ -19,6 +25,10 @@ dev = [
     "mypy>=1.14.0",
     "azure-identity>=1.19.0",
     "black>=24.10.0",
+    "types-tqdm>=4.66.0",
+    "pandas-stubs>=2.1.0",
+    "pydantic-core>=2.16.3",
+    "typing-extensions>=4.9.0"
```

Comment:
NIT: Why required at root level?

Reply:
moving upper to to `evals`, removing lower 2. I got confused sorry.